### ISSUE TITLE
Add open pull requests to agenda page

### ISSUE THREAD

**Issue by krzyk** [zerocracy/farm#1159]
Right now it is hard to follow which tasks I have pushed PRs for and which are not started (or I want to look if travis build passed in all my PRs).
The solution would be to find the PRs for each of the assigned tasks and show it inside another column (named `PR`).

This should be simple, one needs to look for open PRs in given repo(s) and find the ones that have title starting from `#123` where `123` is the number of assigned issue.

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) please, pay attention to this issue

**Comment by g4s8** [comment]
@krzyk right, it's hard to track all PRs and open issues, but I don't think that agenda is good place for it, because it includes only tasks assigned to you. What do you think about new page with tasks in WBS authored by you? 

@amihaiemil @carlosmiranda @proshin-roman please, look at this issue also.

**Comment by amihaiemil** [comment]
@g4s8 Yes, a page with tasks authored by the user is the only option, I would say. 

Because currently 0crat has no session for Issues -- it cannot easily correlate an Issue with an open PR, unless we make some kind of convention which will surely be broken sooner or later and introduce exceptions, bugs etc.

Another solution would be to have 0crat add badges to the Issues: it could label the Issues assigned to me with my username and then I can easily filter out the Issues which are assigned to me and see if I made a PR for them or not.

**Comment by krzyk** [comment]
@g4s8 sounds good to me

**Comment by krzyk** [comment]
@g4s8 I updated the description.
This will solve the problem partially, but maybe it will be enough (at least until github introduces some nice way of linking PRs with issues).

**Comment by 0crat** [comment]
Job [#1159](https://github.com/zerocracy/farm/issues/1159) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 point(s) just awarded to @krzyk[/z](https://www.0crat.com/u/krzyk)

**Comment by 0pdd** [comment]
@krzyk 2 puzzles [#1640](https://github.com/zerocracy/farm/issues/1640), [#1641](https://github.com/zerocracy/farm/issues/1641) are still not solved.

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @krzyk[/z](https://www.0crat.com/u/krzyk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#1159](https://github.com/zerocracy/farm/issues/1159) is now out of scope

**Comment by ypshenychka** [comment]
@0crat quality goof

**Comment by ypshenychka** [comment]
@0crat quality good

**Comment by 0crat** [comment]
Order was finished, quality is "good": +35 point(s) just awarded to @krzyk[/z](https://www.0crat.com/u/krzyk)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Comment by 0pdd** [comment]
@krzyk 2 puzzles [#1641](https://github.com/zerocracy/farm/issues/1641), [#2279](https://github.com/zerocracy/farm/issues/2279) are still not solved; solved: [#1640](https://github.com/zerocracy/farm/issues/1640).

**Comment by 0pdd** [comment]
@krzyk the puzzle [#unknown]() is still not solved.
